### PR TITLE
Add SM-T355Y

### PR DESCRIPTION
**Pull request by hello-world-321**
Adds Samsung Tab A 8.0 (msm8916) (gt58lte-eur)

### CODE REVIEW

File: dts/msm8916/msm8916-samsung-r04.dts [hunk 1]
```diff
@@ -19,4 +19,10 @@
 			i2c-address = <0x25>;
 		};
 	};
+	
+	gt58lte-eur {
+                model = "Samsung Galaxy Tab A 8.0 2015 (SM-T355Y)";
+                compatible = "samsung,gt58lte-eur", "qcom,msm8916", "lk2nd,device";
```
Comment (Minecrell) [inline dts/msm8916/msm8916-samsung-r04.dts:25]: As far as I can tell, the SM-T355Y is the Australian variant and is `msm8916-sec-gt58lte-aus-r00.dtsi`(i.e. `samsung,gt58lte-aus`) downstream. Do you have the Australian variant maybe?

**Review by hello-world-321 — COMMENTED** [review]
Changed the variant.

**Comment by TravMurav** [comment]
Please make single commit with all the changes, make the commit description a bit more verbose (look how it was done for other devices), preferably use git and not the web ui. Indention is tabs and not spaces

**Comment by hello-world-321** [comment]
> Please make single commit with all the changes, make the commit description a bit more verbose (look how it was done for other devices), preferably use git and not the web ui. Indention is tabs and not spaces
> 
> Also avoid using master branch for new features even on your fork

Done

File: dts/msm8916/msm8916-samsung-r04.dts [hunk 2]
```diff
@@ -19,4 +19,10 @@
 			i2c-address = <0x25>;
 		};
 	};
+	
+	gt58lte-aus {
+                model = "Samsung Galaxy Tab A 8.0 2015 (SM-T355Y)";
```
Comment (TravMurav) [inline dts/msm8916/msm8916-samsung-r04.dts:24]: Indention is tabs and not spaces

**Review by TravMurav — CHANGES_REQUESTED** [review]

**Review by hello-world-321 — COMMENTED** [review]
Will use vscode to fix it tomorrow.

File: dts/msm8916/msm8916-samsung-r04.dts (hunk 2 above)
Comment (hello-world-321) [reply dts/msm8916/msm8916-samsung-r04.dts]: Resolved

**Comment by hello-world-321** [comment]
Fixed

**Review by TravMurav — APPROVED** [review]

**Pull request merged**